src/components/ui-language-dialog.less

```diff
@@ -0,0 +1,24 @@
+.UiLanguageDialog {}
```

Comment:
Is this really needed?

Reply:
We always include it to have the Component part of SUITCSS there, it's compiled away anyway by the optimizer.